src/org/zaproxy/zap/authentication/PostBasedAuthenticationMethodType.java

```diff
@@ -315,6 +321,51 @@ public WebSession authenticate(SessionManagementMethod sessionManagementMethod,
 			// Return the web session as extracted by the session management method
 			return sessionManagementMethod.extractWebSession(msg);
 		}
+		
+		/**
+		 * Modifies the input {@code requestMessage} by replacing fresh anti-CSRF token value in the request body.  
+		 * It first checks if the input {@code requestMessage} has an anti-CSRF token.
+		 * If yes, then it makes a {@code GET} request to the login page in order to renew the anti-CSRF token.
+		 * Then it modifies the input {@code requestMessage} with the regenerated anti-CSRF token value.
+		 * If the first {@code POST} login request does not have anti-CSRF token
+		 * then the input {@code requestMessage} is left as it is.
+		 * 
+		 * @param	requestMessage	the login request message with correct credential
```

Comment:
It does not worth align the parameters (and their descriptions), that adds unnecessary work (to keep them aligned when they change, also it does not make any difference for rendered JavaDoc).